### PR TITLE
homework

### PR DESCRIPTION
**Pull request by Daniluko**

### CODE REVIEW

File: src/test/java/project/springData/service/OfficesServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    @Test
+    public void testFindOfficeById() {
+        doReturn(Optional.of(office1)).when(officesRepository).findById(any());
+        Office result = officesService.findOfficeById(BigDecimal.ONE);
+        assertEquals(office1, result);
+        System.out.println(result);
+    }
+
+    @Test
+    public void testInsertOffice() {
```
Comment (SerhiiPasko) [inline src/test/java/project/springData/service/OfficesServiceImplTest.java:63]: Tests not full. You also must test when , in this case, office already exist

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Read comment

File: src/main/java/project/controller/OfficeController.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            LOG.debug("This method use method GetAllOffices");
+            return officesService.getAllOffice();
+        }
+        Set<Office> offices = officesService.findOfficeFirstByTargetAfter(target);
+        LOG.info("Office found : ");
+        return offices;
+    }
+
+    @PutMapping("/{id}")
+    public @ResponseBody
```
Comment (Yanetta) [inline src/main/java/project/controller/OfficeController.java]: Does this response have body?

**Review by SerhiiPasko — APPROVED** [review]
Good

**Pull request merged**